propose a trivial solution (brute force)

borgesgfj opened the pull request:
- Problem overview:
	-Given a sorted list of numbers and a certain number of queries
	to be performed, return the index of the  first occurrence of a query
	number if the array contains it, in other case return -1.
- Solution overview:
	-Querie numbers and the list number are declared in a array of
	objects as arrays;
	-The result is pushed in an array contains the index of each
	arrayNumbers element requested by a querie element

borgesgfj committed:
propose a trivial solution (brute force)

- Problem overview:
	-Given a sorted list of numbers and a certain number of queries
	to be performed, return the index of the  first occurrence of a query
	number if the array contains it, in other case return -1.
- Solution overview:
	-Querie numbers and the list number are declared in a array of
	objects as arrays;
	-The result is pushed in an array contains the index of each
	arrayNumbers element requested by a querie element

cassios commented on binary-search/brute-force-solution.js:
```diff
@@ -0,0 +1,30 @@
+  let resultsQueries = []
+  for(let j = 0; j < listOfQueries.length; j++){
+    let elementIndex = -1;
+    resultsQueries.push(elementIndex)
```
ser vc mudar a inserção da resposta pra depois do for, vc não vai precisar modificar a lista de respostas caso encontre o valor

borgesgfj replied:
Boa mano. Valeu fica mais enxuto!

borgesgfj committed:
Implement binary search alghorithm for first occurrence.

- A variant of binary search for find the first occurrence of an element
  was implemented;
- More explanations about this algorithm was given in the .md file
  consideracoes-buscaBinaria, where the basics was explained and some
  references are given;
- In the proposed solution the max and min variables refered in
  consideracoes-buscaBinaria.md are named as maxIdex and minIndex
  respectively;
- The problem statment is given on file enunciado.md

borgesgfj committed:
Change the array resultQueries implement to be after the inner for

cassios reviewed (approved):
infinitos parabéns 👏🏽 👏🏽
